Common/Systems/DisableBuilding/StopBuildingPlayer.cs

```diff
@@ -106,7 +106,7 @@ public override void ResetEffects()
 
 		if (SubworldSystem.Current is MappingWorld)
 		{
-			ConstantStopBuilding = true;
+			//ConstantStopBuilding = true;
```

Comment:
Unintended?

Reply:
Whoops! Yeah, I was building in Ravencrest and forgot to undo this.